vllm/engine/llm_engine.py

```diff
@@ -140,7 +140,10 @@ def get_tokenizer_for_seq(self, sequence: Sequence):
     def _init_workers(self):
         # Lazy import the Worker to avoid importing torch.cuda/xformers
         # before CUDA_VISIBLE_DEVICES is set in the Worker
-        from vllm.worker.worker import Worker
+        if is_neuron():
```

Comment:
nit: Can we cache `self.is_neuron = is_neuron()` just like what we did for model runner?

Reply:
Sure, i'm a big fan of hashing and caching. It's now cached as in model runner.